tools/pipelines/templates/build-npm-package.yml

```diff
@@ -427,6 +458,15 @@ stages:
               targetPath: '$(Build.ArtifactStagingDirectory)/test-files'
               artifactName: 'test-files'
               publishLocation: 'pipeline'
+        
+        # Point types back to default
+        - ${{ if or(eq(parameters.releaseBuildOverride, 'prerelease - alpha'), eq(parameters.releaseBuildOverride, 'prerelease - beta')) }}:
```

Comment:
Another thing that can be simplified once you use alpha/beta. This step can just always run in the same if block as the other one.